docs/OpenShiftPipelinesAsCode.md

```diff
@@ -0,0 +1,68 @@
+<!--
+---
+linkTitle: "OpenShiftPipelinesAsCode"
+weight: 9
+---
+-->
+# OpenShiftPipelinesAsCode
+
+OpenShiftPipelinesAsCode custom resource allows user to install and manage [OpenShift PipelinesAsCode](PipelinesAsCode).
+
+It is recommended to install the component through [TektonConfig](./TektonConfig.md).
+
+- OpenShiftPipelinesAsCode CR is as below
+
+    - On OpenShift, OpenShiftPipelinesAsCode CR is as below:
+
+    ```yaml
+    apiVersion: operator.tekton.dev/v1alpha1
+    kind: OpenShiftPipelinesAsCode
+    metadata:
+      name: pipelines-as-code
+    spec:
+      settings:
+        application-name: Pipelines as Code CI
+        auto-configure-new-github-repo: "false"
+        bitbucket-cloud-check-source-ip: "true"
+        custom-console-name: ""
+        custom-console-url: ""
+        custom-console-url-pr-details: ""
+        custom-console-url-pr-tasklog: ""
+        error-detection-from-container-logs: "false"
+        error-detection-max-number-of-lines: "50"
+        error-detection-simple-regexp: ^(?P<filename>[^:]*):(?P<line>[0-9]+):(?P<column>[0-9]+):([
+          ]*)?(?P<error>.*)
+        error-log-snippet: "true"
+        hub-catalog-name: tekton
+        hub-url: https://api.hub.tekton.dev/v1
+        remote-tasks: "true"
+        secret-auto-create: "true"
+        secret-github-app-token-scoped: "true"
+      targetNamespace: openshift-pipelines
+    ```
+
+- Check the status of installation using following command:
+
+    ```sh
+    kubectl get openshiftpipelinesascodes.operator.tekton.dev
+    ```
+
+## PipelinesAsCode Config 
+
+There are some fields which you can define on OpenShiftPipelinesAsCode CR to configure the behaviour of the PipelinesAsCode
```

Comment:
Probably this is not the expected way and one should use TektonConfig instead?

Reply:
Yes you are right modified sentence PTAL
Thank you